src/zephyr_coap_req.c

```diff
@@ -699,10 +701,12 @@ static int64_t golioth_coap_req_poll_prepare(struct golioth_coap_req *req, uint3
     {
         if (resend)
         {
-            LOG_WRN("Resending request %p (reply %p) (retries %d)",
+            LOG_DBG("Resending request %p (reply %p) (retries %d)",
```

Comment:
This should still be a warning.

Reply:
I think you meant to leave this comment on line 722, is that right?

```
LOG_INF("%u resends in last %d seconds",
```